applications/luci-app-speedtest/root/usr/share/rpcd/acl.d/luci-app-speedtest.json

```diff
@@ -0,0 +1,10 @@
+{
+	"luci-app-speedtest": {
+		"description": "Grant access to speedtest",
+		"write": {
+			"file": {
+				"/usr/bin/speedtest": [ "exec"]
```

Comment:
```suggestion
				"/usr/bin/speedtest": ["exec"]
```

Reply:
OK, I thought I had left spaces when I wrote, but apparently not